common/lib/xmodule/xmodule/modulestore/split_mongo/split_draft.py

```diff
@@ -58,7 +58,11 @@ def get_course(self, course_id, depth=0, **kwargs):
         course_id = self._map_revision_to_branch(course_id)
         return super(DraftVersioningModuleStore, self).get_course(course_id, depth=depth, **kwargs)
 
-    def get_library(self, library_id, depth=0, **kwargs):
+    def get_library(self, library_id, depth=0, head_validation=True, **kwargs):
+        if not head_validation and library_id.version_guid:
+            return SplitMongoModuleStore.get_library(
+                self, library_id, depth=depth, head_validation=head_validation, **kwargs
+            )
```

Comment:
Isn't `SplitMongoModuleStore.get_library` the same as ` super(DraftVersioningModuleStore, self).get_library` in this context? It is as far as I can see. In which case, I think this would look nicer if you just left `head_validation` hidden inside `**kwargs` and reverted all your changes to this method, to keep it simple.